.github/workflows/buf.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: wizhi/setup-buf@v1
         with:
-          version: 0.20.5
+          version: 1.47.2
```

Comment:
should the linter be dead after this commit? it is now

Reply:
I'll throw away these commits probably, this action doesn't work with the new version, "official" action requires yet another registration/token. The biggest problem is configuration format change, current `buf` doesn't accept old config.

Reply:
I've left it. This means that compatibility check will fail for this PR specifically because we're trying to use the new `buf` with the old config from master in the check and this can't work. This check is also somewhat suspicious, with fixed config it yields
```
master/accounting/types.proto:15:9:symbol "neo.fs.v2.accounting.Decimal" already defined at accounting/types.proto:15:9
master/accounting/types.proto:17:9:symbol "neo.fs.v2.accounting.Decimal.value" already defined at accounting/types.proto:17:9
master/accounting/types.proto:21:10:symbol "neo.fs.v2.accounting.Decimal.precision" already defined at accounting/types.proto:21:10
refs/types.proto:12:9:symbol "neo.fs.v2.refs.Address" already defined at master/refs/types.proto:12:9
refs/types.proto:14:15:symbol "neo.fs.v2.refs.Address.container_id" already defined at master/refs/types.proto:14:15
refs/types.proto:16:12:symbol "neo.fs.v2.refs.Address.object_id" already defined at master/refs/types.proto:16:12
refs/types.proto:36:9:symbol "neo.fs.v2.refs.ObjectID" already defined at master/refs/types.proto:36:9
refs/types.proto:38:9:symbol "neo.fs.v2.refs.ObjectID.value" already defined at master/refs/types.proto:38:9
refs/types.proto:57:9:symbol "neo.fs.v2.refs.ContainerID" already defined at master/refs/types.proto:57:9
refs/types.proto:59:9:symbol "neo.fs.v2.refs.ContainerID.value" already defined at master/refs/types.proto:59:9
refs/types.proto:77:9:symbol "neo.fs.v2.refs.OwnerID" already defined at master/refs/types.proto:77:9
refs/types.proto:79:9:symbol "neo.fs.v2.refs.OwnerID.value" already defined at master/refs/types.proto:79:9
refs/types.proto:89:9:symbol "neo.fs.v2.refs.SubnetID" already defined at master/refs/types.proto:89:9
refs/types.proto:91:11:symbol "neo.fs.v2.refs.SubnetID.value" already defined at master/refs/types.proto:91:11
refs/types.proto:98:9:symbol "neo.fs.v2.refs.Version" already defined at master/refs/types.proto:98:9
refs/types.proto:100:10:symbol "neo.fs.v2.refs.Version.major" already defined at master/refs/types.proto:100:10
refs/types.proto:103:10:symbol "neo.fs.v2.refs.Version.minor" already defined at master/refs/types.proto:103:10
refs/types.proto:107:9:symbol "neo.fs.v2.refs.Signature" already defined at master/refs/types.proto:107:9
refs/types.proto:109:9:symbol "neo.fs.v2.refs.Signature.key" already defined at master/refs/types.proto:109:9
refs/types.proto:111:9:symbol "neo.fs.v2.refs.Signature.sign" already defined at master/refs/types.proto:111:9
refs/types.proto:113:19:symbol "neo.fs.v2.refs.Signature.scheme" already defined at master/refs/types.proto:113:19
refs/types.proto:117:6:symbol "neo.fs.v2.refs.SignatureScheme" already defined at master/refs/types.proto:117:6
refs/types.proto:119:3:symbol "neo.fs.v2.refs.ECDSA_SHA512" already defined at master/refs/types.proto:119:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
refs/types.proto:122:3:symbol "neo.fs.v2.refs.ECDSA_RFC6979_SHA256" already defined at master/refs/types.proto:122:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
refs/types.proto:126:3:symbol "neo.fs.v2.refs.ECDSA_RFC6979_SHA256_WALLET_CONNECT" already defined at master/refs/types.proto:126:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
refs/types.proto:130:9:symbol "neo.fs.v2.refs.SignatureRFC6979" already defined at master/refs/types.proto:130:9
refs/types.proto:132:9:symbol "neo.fs.v2.refs.SignatureRFC6979.key" already defined at master/refs/types.proto:132:9
refs/types.proto:134:9:symbol "neo.fs.v2.refs.SignatureRFC6979.sign" already defined at master/refs/types.proto:134:9
refs/types.proto:138:6:symbol "neo.fs.v2.refs.ChecksumType" already defined at master/refs/types.proto:138:6
refs/types.proto:140:3:symbol "neo.fs.v2.refs.CHECKSUM_TYPE_UNSPECIFIED" already defined at master/refs/types.proto:140:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
refs/types.proto:143:3:symbol "neo.fs.v2.refs.TZ" already defined at master/refs/types.proto:143:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
refs/types.proto:146:3:symbol "neo.fs.v2.refs.SHA256" already defined at master/refs/types.proto:146:3; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
refs/types.proto:156:9:symbol "neo.fs.v2.refs.Checksum" already defined at master/refs/types.proto:156:9
refs/types.proto:158:16:symbol "neo.fs.v2.refs.Checksum.type" already defined at master/refs/types.proto:158:16
refs/types.proto:161:9:symbol "neo.fs.v2.refs.Checksum.sum" already defined at master/refs/types.proto:161:9
status/types.proto:35:9:symbol "neo.fs.v2.status.Status" already defined at master/status/types.proto:35:9
status/types.proto:37:10:symbol "neo.fs.v2.status.Status.code" already defined at master/status/types.proto:37:12
status/types.proto:40:10:symbol "neo.fs.v2.status.Status.message" already defined at master/status/types.proto:40:12
status/types.proto:46:11:symbol "neo.fs.v2.status.Status.Detail" already defined at master/status/types.proto:46:13
status/types.proto:49:12:symbol "neo.fs.v2.status.Status.Detail.id" already defined at master/status/types.proto:49:16
status/types.proto:53:11:symbol "neo.fs.v2.status.Status.Detail.value" already defined at master/status/types.proto:53:15
status/types.proto:57:19:symbol "neo.fs.v2.status.Status.details" already defined at master/status/types.proto:57:21
status/types.proto:61:6:symbol "neo.fs.v2.status.Section" already defined at master/status/types.proto:61:6
status/types.proto:63:3:symbol "neo.fs.v2.status.SECTION_SUCCESS" already defined at master/status/types.proto:63:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:66:3:symbol "neo.fs.v2.status.SECTION_FAILURE_COMMON" already defined at master/status/types.proto:66:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:69:3:symbol "neo.fs.v2.status.SECTION_OBJECT" already defined at master/status/types.proto:69:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:72:3:symbol "neo.fs.v2.status.SECTION_CONTAINER" already defined at master/status/types.proto:72:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:75:3:symbol "neo.fs.v2.status.SECTION_SESSION" already defined at master/status/types.proto:75:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:79:6:symbol "neo.fs.v2.status.Success" already defined at master/status/types.proto:79:6
status/types.proto:83:3:symbol "neo.fs.v2.status.OK" already defined at master/status/types.proto:83:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:87:6:symbol "neo.fs.v2.status.CommonFail" already defined at master/status/types.proto:87:6
status/types.proto:91:3:symbol "neo.fs.v2.status.INTERNAL" already defined at master/status/types.proto:91:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:97:3:symbol "neo.fs.v2.status.WRONG_MAGIC_NUMBER" already defined at master/status/types.proto:97:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:100:3:symbol "neo.fs.v2.status.SIGNATURE_VERIFICATION_FAIL" already defined at master/status/types.proto:100:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:103:3:symbol "neo.fs.v2.status.NODE_UNDER_MAINTENANCE" already defined at master/status/types.proto:103:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:107:6:symbol "neo.fs.v2.status.Object" already defined at master/status/types.proto:107:6
status/types.proto:111:3:symbol "neo.fs.v2.status.ACCESS_DENIED" already defined at master/status/types.proto:111:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:114:3:symbol "neo.fs.v2.status.OBJECT_NOT_FOUND" already defined at master/status/types.proto:114:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:117:3:symbol "neo.fs.v2.status.LOCKED" already defined at master/status/types.proto:117:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:120:3:symbol "neo.fs.v2.status.LOCK_NON_REGULAR_OBJECT" already defined at master/status/types.proto:120:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:123:3:symbol "neo.fs.v2.status.OBJECT_ALREADY_REMOVED" already defined at master/status/types.proto:123:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:126:3:symbol "neo.fs.v2.status.OUT_OF_RANGE" already defined at master/status/types.proto:126:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:130:6:symbol "neo.fs.v2.status.Container" already defined at master/status/types.proto:130:6
status/types.proto:132:3:symbol "neo.fs.v2.status.CONTAINER_NOT_FOUND" already defined at master/status/types.proto:132:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:135:3:symbol "neo.fs.v2.status.EACL_NOT_FOUND" already defined at master/status/types.proto:135:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:139:6:symbol "neo.fs.v2.status.Session" already defined at master/status/types.proto:139:6
status/types.proto:141:3:symbol "neo.fs.v2.status.TOKEN_NOT_FOUND" already defined at master/status/types.proto:141:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
status/types.proto:144:3:symbol "neo.fs.v2.status.TOKEN_EXPIRED" already defined at master/status/types.proto:144:5; protobuf uses C++ scoping rules for enum values, so they exist in the scope enclosing the enum
```
But it's not related to these changes. Let's see how it work in future PRs.